content/manuals/build/concepts/dockerfile.md

```
@@ -112,46 +103,40 @@ Here's a breakdown of what this Dockerfile does:
 
 ### Dockerfile syntax
 
-The first line to add to a Dockerfile is a [`# syntax` parser directive](/reference/dockerfile.md#syntax).
-While optional, this directive instructs the Docker builder what syntax to use
-when parsing the Dockerfile, and allows older Docker versions with [BuildKit enabled](../buildkit/_index.md#getting-started)
-to use a specific [Dockerfile frontend](../buildkit/frontend.md) before
-starting the build. [Parser directives](/reference/dockerfile.md#parser-directives)
-must appear before any other comment, whitespace, or Dockerfile instruction in
-your Dockerfile, and should be the first line in Dockerfiles.
+The first line is a [`# syntax` parser directive](/reference/dockerfile.md#syntax).
+This optional directive tells Docker which syntax to use when parsing the Dockerfile.
+It lets older Docker versions with [BuildKit enabled](../buildkit/_index.md#getting-started)
+use a specific [Dockerfile frontend](../buildkit/frontend.md) before starting the
+build. [Parser directives](/reference/dockerfile.md#parser-directives) must appear
+before any other comment, whitespace, or instruction, and should be the first line.
 
 ```dockerfile
 # syntax=docker/dockerfile:1
 ```
 
 > [!TIP]
->
-> We recommend using `docker/dockerfile:1`, which always points to the latest
-> release of the version 1 syntax. BuildKit automatically checks for updates of
-> the syntax before building, making sure you are using the most current version.
+> Use `docker/dockerfile:1` to always get the latest version 1 syntax. BuildKit
+> checks for updates before building, so you use the most current version.
 
 ### Base image
 
-The line following the syntax directive defines what base image to use:
+The next line defines the base image:
 
 ```dockerfile
 FROM ubuntu:22.04
 ```
 
 The [`FROM` instruction](/reference/dockerfile.md#from) sets your base
-image to the 22.04 release of Ubuntu. All instructions that follow are executed
-in this base image: an Ubuntu environment. The notation `ubuntu:22.04`, follows
-the `name:tag` standard for naming Docker images. When you build images, you
-use this notation to name your images. There are many public images you can
-leverage in your projects, by importing them into your build steps using the
-Dockerfile `FROM` instruction.
+image to Ubuntu 22.04. All following instructions run in this Ubuntu environment. The
```

This assume people know what means 22.04 for Ubuntu which might not be the case for introducing to concepts. I think we should keep the current copy:

```suggestion
image to the 22.04 release of Ubuntu. All following instructions run in this Ubuntu environment. The
```